lib/aptible/cli/helpers/operation.rb

```diff
@@ -76,11 +75,10 @@ def get_operation_logs_redirect(operation)
           http = Net::HTTP.new(uri.host, uri.port)
           http.use_ssl = true
           res = http.request(Net::HTTP::Get.new(uri.request_uri, headers))
-          # note: res :location is the  header "location" for a 301
+          # note: res body with a 200 is target redirect location for download
           # in approx ruby 2.4 or lower, location is in a different spot
```

Comment:
Comment is no longer applicable

Reply:
dropping the 2nd comment, but i updated the first i think when this was reviewed to be somewhat accurate